depthai_demo.py

```diff
@@ -10,21 +10,23 @@
 from pathlib import Path
 
 import cv2
-os.environ["DEPTHAI_INSTALL_SIGNAL_HANDLER"] = "0"
+
 import depthai as dai
```

Comment:
Let's add try except for dependency packages (e.g. Cv2, numpy, depthai).
This can result in failure if the user runs only the demo, without runnimg install requirements beforehand

Reply:
just added - https://github.com/luxonis/depthai/pull/573/commits/081d7f3321563da09009378f0605c36c0b6bde2b

Reply:
Also pointed user to our install instructions in the docs in case something goes wrong - https://github.com/luxonis/depthai/pull/573/commits/f24f10a5505d79d22c25e085e34cb2b5d78209a2